### PR TITLE
Added an endpoint for adding recipients to a batch invoice. Slight refactor on TSDKInvoiceStatus

### PR DESCRIPTION
**Pull request by jrahaim**

### CODE REVIEW

File: TeamSnapSDK/SDK/DataTypes/TSDKInvoice.m [hunk 1]
```diff
@@ -33,18 +33,40 @@ - (CGFloat)percentPaid {
...
     } else {
         return TSDKInvoiceStatusUnknown;
     }
 }
 
++(NSString *_Nonnull)invoiceStatusStringForStatus:(TSDKInvoiceStatus)status {
+    switch (status) {
+        case TSDKInvoiceStatusOpen:
+            return @"open";
+            break;
```
Comment (larsacus) [inline TeamSnapSDK/SDK/DataTypes/TSDKInvoice.m]: this break will never execute and may trigger a dead code warning

File: TeamSnapSDK/SDK/DataTypes/TSDKInvoice.m [hunk 2]
```diff
@@ -33,18 +33,40 @@ - (CGFloat)percentPaid {
...
+        case TSDKInvoiceStatusPaid:
+            return @"paid";
+            break;
+        case TSDKInvoiceStatusCanceled:
+            return @"canceled";
+            break;
+        default:
```
Comment (larsacus) [inline TeamSnapSDK/SDK/DataTypes/TSDKInvoice.m]: If this switch is exhaustive, we shouldn't need a default case

Comment (jrahaim) [reply TeamSnapSDK/SDK/DataTypes/TSDKInvoice.m]: It wasn't exhaustive, but it should have been to make it future proof! Fixed.

File: TeamSnapSDK/SDK/DataTypes/TSDKInvoice.m (hunk 1 above)
Comment (jrahaim) [reply TeamSnapSDK/SDK/DataTypes/TSDKInvoice.m]: Fixed.

**Comment by jrahaim** [comment]
Investigating the compiler errors.

**Comment by jrahaim** [comment]
(IDK why Xcode popped up, then removed a ton of compiling errors. All good)

**Comment by larsacus** [comment]
This looks good 👍

**Pull request merged**